includes/admin/settings/register-settings.php

```diff
@@ -1293,12 +1293,12 @@ function edd_settings_sanitize( $input = array() ) {
 					}
 
 					$setting_details = edd_get_registered_setting_details( $tab, $section, $key );
-					$number_type = false !== strpos( $setting_details['step'], '.' ) ? 'floatval' : 'intval';
-					$minimum   = $number_type( $setting_details['min'] );
-					$maximum   = $number_type( $setting_details['max'] );
-					$new_value = $number_type( $input[ $key ] );
+					$number_type     = ! empty( $setting_details['step'] ) && false !== strpos( $setting_details['step'], '.' ) ? 'floatval' : 'intval';
+					$minimum         = ! empty( $setting_details['min'] ) ? $number_type( $setting_details['min'] ) : false;
+					$maximum         = ! empty( $setting_details['max'] ) ? $number_type( $setting_details['max'] ) : false;
+					$new_value       = $number_type( $input[ $key ] );
 
-					if ( $minimum > $new_value || $maximum < $new_value ) {
+					if ( ( $minimum && $minimum > $new_value ) || ( $maximum && $maximum < $new_value ) ) {
```

Comment:
Do we need to be explicitly checking `false` here? Example:

instead of `if ( $minimum ... )`
do: `if ( false !== $minimum ... )`

Just think that the min/max could be `0`